books/free-programming-books-langs.md

```diff
@@ -403,6 +404,11 @@ That section got so big, we decided to split it into its own file, the [BY SUBJE
 * [Object-Oriented Programming in the BETA Programming Language](https://beta.cs.au.dk/Books/) - Ole Lehrmann Madsen, Birger Møller-Pedersen, Kristen Nygaard
 
 
+### BeanShell
+
+* [Beanshell Simple Java Scripting Manual](http://www.beanshell.org/manual/bshmanual.pdf)
```

Comment:
Please, try to:
- Satify linter pdf links rule.
  ```markdown
  * [Beanshell Simple Java Scripting Manual](http://www.beanshell.org/manual/bshmanual.pdf) (PDF)
  ```
- Give authoring attributions: beanshell.org, Patrick Niemeyer (as first creator)
- There are also an online version. Maybe use multiple format syntax.

```markdown
* [Beanshell Simple Java Scripting Manual](http://www.beanshell.org/manual/bshmanual.pdf) - beanshell.org, Patrick Niemeyer (HTML, [PDF](http://www.beanshell.org/manual/bshmanual.pdf))
```